.flaskenv

```diff
@@ -0,0 +1 @@
+export FLASK_DEBUG=False
```

Comment:
Previously, I used CLI to set the value of FLASK_DEBUG. I thought that it will be better to set it in a .flaskenv file. I understand that is it not a good practice?

Reply:
https://github.com/Jaqbek95/hops_flask_api/pull/3#discussion_r1114300744

Reply:
fixed